return {
	[1] = {
		["type"] = 1,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 1,
		["price1"] = 19,
		["name_t"] = "Beginner Pack",
		["bgIconId"] = "rbxassetid://77886307507247",
		["price2"] = 0,
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price3"] = 0,
		["canRepurchased"] = 1,
		["beforeDiscountPrice3"] = 0,
		["productId3"] = 0,
		["sortId"] = 1,
		["productId1"] = 3567697645,
	},
	[2] = {
		["type"] = 1,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 2,
		["price1"] = 49,
		["name_t"] = "Pro Pack",
		["bgIconId"] = "rbxassetid://83983663285653",
		["price2"] = 0,
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price3"] = 0,
		["canRepurchased"] = 1,
		["beforeDiscountPrice3"] = 0,
		["productId3"] = 0,
		["sortId"] = 2,
		["productId1"] = 3567697940,
	},
	[3] = {
		["type"] = 2,
		["productId2"] = 3567699762,
		["beforeDiscountPrice2"] = 297,
		["id"] = 3,
		["price1"] = 99,
		["name_t"] = "Sprite Chest",
		["bgIconId"] = "rbxassetid://123495708111145",
		["price2"] = 249,
		["nameColor"] = {
			[1] = 85,
			[2] = 255,
			[3] = 127,
		},
		["price3"] = 888,
		["canRepurchased"] = 1,
		["beforeDiscountPrice3"] = 990,
		["productId3"] = 3567699828,
		["sortId"] = 3,
		["productId1"] = 3567699692,
	},
	[4] = {
		["type"] = 3,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 4,
		["price1"] = 185,
		["name_t"] = "Double Jump Coil",
		["bgIconId"] = "rbxassetid://107017979973801",
		["price2"] = 0,
		["nameColor"] = {
			[1] = 164,
			[2] = 139,
			[3] = 219,
		},
		["price3"] = 0,
		["canRepurchased"] = 0,
		["beforeDiscountPrice3"] = 0,
		["productId3"] = 0,
		["sortId"] = 4,
		["productId1"] = 3567698503,
	},
	[5] = {
		["type"] = 3,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 5,
		["price1"] = 149,
		["name_t"] = "Gravity Coil",
		["bgIconId"] = "rbxassetid://80037247271171",
		["price2"] = 0,
		["nameColor"] = {
			[1] = 164,
			[2] = 139,
			[3] = 219,
		},
		["price3"] = 0,
		["canRepurchased"] = 0,
		["beforeDiscountPrice3"] = 0,
		["productId3"] = 0,
		["sortId"] = 5,
		["productId1"] = 3567698743,
	},
}